lib/openstack_api/openstack_query.py

```diff
@@ -272,6 +272,16 @@ def get_user_prop(user_id, prop):
             }
         if object_type == "user":
             return {}
+        if object_type == "hypervisor":
+            return {
+                # Note: These parameters are depreciated, currently the openstack sdk is using a max microversion
+                # 2.88 meaning these will work but the 'uptime' parameter will not. Later on we will need
```

Comment:
Since `uptime` is unavailable at our current SDK version does this mean we cannot query HV uptimes?
If not can we raise a `NotImplementedError` when a user tries to use that query

Reply:
Correct, it just returns None. Good idea, thanks.